### PR TITLE
N21-1645 refactor tool element

### PR DESCRIPTION
**Pull request by IgorCapCoder**
# Short Description
this refactors the tool-element of the column board by using the content-element-bar component.

## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/N21-1645

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/feature-board-external-tool-element/ExternalToolElement.vue [hunk 1]
```diff
@@ -260,11 +270,6 @@ $logo-size: 24px;
 	padding: $card-padding;
```
Comment (MarvinOehlerkingCap) [inline src/components/feature-board-external-tool-element/ExternalToolElement.vue:270]: the styles and variables are not used anymore

File: src/components/feature-board-external-tool-element/ExternalToolElement.vue [hunk 2]
```diff
@@ -13,35 +13,35 @@
...
-				ref="externalToolElementMenu"
-				@move-down:element="onMoveElementDown"
-				@move-up:element="onMoveElementUp"
-				@delete:element="onDeleteElement"
-				@edit:element="onEditElement"
-			/>
-		</div>
+			</template>
+			<template #menu>
+				<slot>
```
Comment (MarvinOehlerkingCap) [inline src/components/feature-board-external-tool-element/ExternalToolElement.vue:33]: why is there a slot here?

File: src/components/feature-board-external-tool-element/ExternalToolElement.unit.ts [hunk 3]
```diff
@@ -551,50 +460,6 @@ describe("ExternalToolElement", () => {
 		});
 	});
 
-	describe("Menu", () => {
```
Comment (MarvinOehlerkingCap) [inline src/components/feature-board-external-tool-element/ExternalToolElement.unit.ts:554]: very unfortunate to lose these tests

**Review by MarvinOehlerkingCap — CHANGES_REQUESTED** [review]

File: src/components/feature-board-external-tool-element/ExternalToolElement.unit.ts (hunk 3 above)
Comment (arnegns) [reply src/components/feature-board-external-tool-element/ExternalToolElement.unit.ts:554]: Yes, but handled over the slot now

File: src/components/feature-board-external-tool-element/ExternalToolElement.vue (hunk 2 above)
Comment (arnegns) [reply src/components/feature-board-external-tool-element/ExternalToolElement.vue:33]: Good point. removed it

File: src/components/feature-board-external-tool-element/ExternalToolElement.vue [hunk 4]
```diff
@@ -251,20 +259,6 @@ export default defineComponent({
...
-	max-width: 100%;
-	min-height: calc($card-padding * 2 + $logo-size);
-	padding: $card-padding;
-}
-
-.logo-container {
-	width: $logo-size;
-	height: $logo-size;
-}
-
```
Comment (MarvinOehlerkingCap) [inline src/components/feature-board-external-tool-element/ExternalToolElement.vue:267]: gap-8, flex-1 and break-word are not used anymore

**Review by MarvinOehlerkingCap — CHANGES_REQUESTED** [review]

File: src/components/feature-board-external-tool-element/ExternalToolElement.vue (hunk 4 above)
Comment (arnegns) [reply src/components/feature-board-external-tool-element/ExternalToolElement.vue:267]: Oh -.- sorry right

**Review by MarvinOehlerkingCap — APPROVED** [review]

**Pull request merged**